src/java/org/apache/cassandra/db/commitlog/CommitLogArchiver.java

```diff
@@ -238,7 +238,7 @@ else if (fromHeader != null)
                     descriptor = fromHeader;
                 else descriptor = fromName;
 
-                if (descriptor.version > CommitLogDescriptor.current_version)
+                if (descriptor.version > CommitLogDescriptor.CURRENT_VERSION)
```

Comment:
nit: Sonar complains that now we need to remove the Visible for testing annotation. Feel free to do it on commit

Reply:
That seems unrelated to the changes, but removing anyway.